File: space-game/OrbitalMechanics/CelestialObjects/Asteroid.vb
Imports Core.Classes
Imports Data.Classes
Imports Entities

Namespace CelestialObjects
    Public Class Asteroid
        Inherits OrbitingCelestialObjectBase

        Public Sub New(name As String,
                       mass As Mass,
                       textures As List(Of Texture),
                       entityManager As IEntityManager,
                       rings As RingData)

            MyBase.New(name, mass, textures, entityManager, rings)
        End Sub

    End Class
End Namespace

File: space-game/OrbitalMechanics/CelestialObjects/Asteroid.cs
using Core.Classes;
using Data.Classes;
using Entities;
using System.Collections.Generic;

namespace OrbitalMechanics.CelestialObjects
{
    public class Asteroid : OrbitingCelestialObjectBase
    {
        public Asteroid(string name, Mass mass, List<Texture> textures, IEntityManager entityManager, RingData rings) : base(name, mass, textures, entityManager, rings)
        {
        }

    }
}
